bubble.rb

```diff
@@ -0,0 +1,23 @@
+def bubble_sort(arr)
+  unsorted_portion = arr.length
+  changes = true
+  while changes  do
+    changes = false
+    (unsorted_portion - 1).times  do |i|
+      puts "arr[i] is #{arr[i]} and arr[i + 1] is #{arr[i+1]}"
+      if arr[i] > arr[i + 1]
+        temp = arr[i + 1]
```

Comment:
To swap 2 elements, `a` and `b` in ruby - `a, b = b, a`.This will eliminate the use of `temp` variable.